azure/scope/managedcontrolplane.go

```diff
@@ -642,14 +664,24 @@ func (s *ManagedControlPlaneScope) MakeEmptyKubeConfigSecret() corev1.Secret {
 	}
 }
 
-// GetKubeConfigData returns a []byte that contains kubeconfig.
-func (s *ManagedControlPlaneScope) GetKubeConfigData() []byte {
-	return s.kubeConfigData
+// GetAdminKubeConfigData returns admin kubeconfig.
+func (s *ManagedControlPlaneScope) GetAdminKubeConfigData() []byte {
+	return s.adminKubeConfigData
+}
+
+// SetAdminKubeConfigData sets adminKubeconfig data.
+func (s *ManagedControlPlaneScope) SetAdminKubeConfigData(kubeConfigData []byte) {
```

Comment:
Hmm what is the change that is suggested here 😅

Not sure because the change suggested and the existing function name both are same. 
SetAdminKubeconfigData(kubeconfigData

Reply:
Ohh the c in Kubeconfig nice catch thanks. ack.